src/test/java/odysseusAT/CharacterizationStepDefs.java

```diff
@@ -0,0 +1,383 @@
+package odysseusAT;
+
+import com.codeborne.selenide.ElementsCollection;
+import com.codeborne.selenide.Selectors;
+import cucumber.api.PendingException;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+import static com.codeborne.selenide.Condition.text;
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selectors.byText;
+import static com.codeborne.selenide.Selenide.*;
+
+public class CharacterizationStepDefs {
+
+    private String generatedString;
+    private String characterizationName;
+    private String generatedStringFeature;
+    private String featureName;
+
+
+    @Then("^can see Characterization page$")
+    public void canSeeCharaterizationPage() {
+        $(By.xpath("//*[@data-bind='text: title']")).shouldHave(text("Cohort Characterizations"));
+    }
+
+    @When("^click to New characterization button$")
+    public void clickToNewCharacterizationButton() {
+        $(By.xpath("//*[@class='characterizations-tabbed-grid__new-entity-btn btn btn-primary btn-sm']")).click();
+        $(By.xpath("//*[@data-bind='text: title']")).shouldHave(text("New Characterization"));
+
+    }
+
+    @When("^enter Characterization name and save it$")
+    public void enterCharacterizationNameAndSaveIt() {
+        generatedString = RandomStringUtils.randomAlphanumeric(10);
+        characterizationName = "Test_" + generatedString;
+        $(By.xpath("//*[@class='input-group']/input")).clear();
+        $(By.xpath("//*[@class='input-group']/input")).setValue(characterizationName);
+        $(By.xpath("//*[@class='fa fa-save']")).click();
+    }
+
+    @When("^return to Characterization table$")
+    public void returnToCharacterizationTable() {
+        $(By.xpath("//*[@class='input-group-btn']/button[2]")).click();
+
+    }
+
+    @When("^enter created characterization name in filter$")
+    public void enterCreatedCharacterizationNameInFilter() {
+        $(By.xpath("//*[@type='search']")).waitUntil(visible, 3000);
+        $(By.xpath("//*[@type='search']")).setValue(characterizationName);
+
+    }
+
+    @Then("^can see new characterization in table$")
+    public void canSeeNewCharacterizationInTable() {
+        $(By.xpath("//*[@class=' characterizations-list__tbl-col characterizations-list__tbl-col--name ']/a")).shouldHave(text(characterizationName));
```

Comment:
Table > Get number of a column which header's title is `Name` > check that links from the column contain the required text

Reply:
What the reason do this?